content/en/docs/tasks/manage-daemon/update-daemon-set.md

```diff
@@ -29,13 +29,13 @@ DaemonSet has two update strategy types:
   DaemonSet template, old DaemonSet pods will be killed, and new DaemonSet pods
   will be created automatically, in a controlled fashion. At most one pod of the DaemonSet will be running on each node during the whole update process.
 
-## Performing a Rolling Update
+## Performing a Rolling Updatec
```

Comment:
is this right?

Reply:
Closing this in favor of https://github.com/kubernetes/website/pull/28319